bigchaindb/pipelines/election.py

```diff
@@ -0,0 +1,51 @@
+import rethinkdb as r
+from multipipes import Pipeline, Node
+
+from bigchaindb.pipelines.utils import ChangeFeed
+from bigchaindb import Bigchain
+
+
+class Election:
+
+    def __init__(self):
+        self.bigchain = Bigchain()
+
+    def check_for_quorum(self, next_vote):
+        """
+        Checks if block has enough invalid votes to make a decision
+        """
+        next_block = r.table('bigchain')\
+            .get(next_vote['vote']['voting_for_block'])\
+            .run(self.bigchain.conn)
+        if self.bigchain.block_election_status(next_block) == self.bigchain.BLOCK_INVALID:
+            return next_block
+
+    def requeue_transactions(self, invalid_block):
+        """
+        Liquidates transactions from invalid blocks so they can be processed again
+        """
+        for tx in invalid_block['block']['transactions']:
+            self.bigchain.write_transaction(tx)
+        return invalid_block
```

Comment:
@vrde just out of curiosity, does the last node in the pipeline need to return anything? or what is the reason for the last node in the pipeline to return?